Mod2/Lessons/Week2/GitForPairs.md

```diff
@@ -0,0 +1,132 @@
+---
+layout:     page
+title:      Git for Pairs
+permalink:  git-for-pairs
+---
+
+# Git for Pairs
+
+## Learning Goals
+  * Outline a workflow for collaborating on a single repository
+  * Identify and resolve merge conflicts
+
+## Vocabulary
+  * Pull Request (PR)
+  * Clone repo
+  * Fork repo
+
+## Review
+> If necessary, review the following video lessons from Version Control with Git/Github
+  >1. [Create A Project and Initialize a Git Repository](https://www.loom.com/share/d3af392e059147a699ce6ac1af65d251)
+  >1. [Add Commits to the Git Repository](https://www.loom.com/share/b7373e5aaad646c790965422719bb993)  
+  >1. [Creating and Pushing to a Github Repository](https://www.loom.com/share/dba188a5f13e43059d9a84b630280c53)
+
+
```

Comment:
I think it might be good to have an intro to paired programming and a review of git and github before diving in to the scenario.  I think it might give students some space to ask any lingering questions they have about the tools and the 'why' before trying things out.

Reply:
@memcmahon Should we use either the [Mod 0 Pair Programming](https://mod0.turing.edu/shared/pairing) or the [BE Mod 1 Pairing Patterns](https://backend.turing.edu/module1/lessons/pairing_patterns) as a guide? Or were you thinking something else?

Reply:
paragraph added under Pair Programming heading